docs/integration-with-android-fragment.md

```diff
@@ -238,6 +230,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
 In the code above `Fragment reactNativeFragment = new ReactFragment.Builder()` creates the ReactFragment and `getSupportFragmentManager().beginTransaction().add()` adds the Fragment to the Frame Layout.
 
+We have to implement the `DefaultHardwareBackBtnHandler` interface and override the `invokeDefaultOnBackPressed` method. This is necessary to allow react-native handle the back button press event.
+
```

Comment:
As per the my understanding from react-native repo [here](https://github.com/facebook/react-native/blob/305b4357eba09347d8f25347c876088bb52ac028/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/core/DefaultHardwareBackBtnHandler.kt#L14), I have added this information. If this seems in correct, feel free to provide the correct one and I will update this PR.

Reply:
What was the crash? I don't think this is strictly necessary

Reply:
It's a requirement per the `ReactDelegate`, otherwise an exception is thrown here:
https://github.com/facebook/react-native/blob/d4d1eb9bb18d66997008988ff0ef48dc754b3fdf/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactDelegate.java#L113-L115